src/Analyser/MutatingScope.php

```diff
@@ -5727,15 +5727,15 @@ public function getIterableValueType(Type $iteratee): Type
 
 	public function getPhpVersion(): PhpVersions
 	{
-		$versionExpr = new ConstFetch(new Name('PHP_VERSION_ID'));
-		if (!$this->hasExpressionType($versionExpr)->yes()) {
```

Comment:
this previous code does not work in files without a namespace, e.g.

```
<?php // lint >= 7.4

if (PHP_VERSION_ID < 80000) {
	\PHPStan\dumpType(PHP_VERSION_ID);
}
```

the `dumpType` returns the correct value, but the scope uses a different expression name

```
$scope->debug();
array (
  '\\PHP_VERSION_ID (Yes)' => 'int<50207, 79999>',
  'native \\PHP_VERSION_ID (Yes)' => 'int<50207, 79999>',
)
```

vs.

```php
<?php // lint >= 7.4

namespace ANamespace;

if (PHP_VERSION_ID < 80000) {
	\PHPStan\dumpType(PHP_VERSION_ID);
}
```

```
$scope->debug();
array (
  'PHP_VERSION_ID (Yes)' => 'int<50207, 79999>',
  'native PHP_VERSION_ID (Yes)' => 'int<50207, 79999>',
)
```

Reply:
Isn't `\PHP_VERSION_ID` const fetch enough?

Reply:
nope, thats the point of this PR